helm/cluster-openstack/templates/kubeadm_control_plane.yaml

```diff
@@ -49,14 +56,20 @@ spec:
         name: '{{ `{{ local_hostname }}` }}'
     useExperimentalRetryJoin: true
     files:
-      {{- include "sshFiles" . | nindent 10 }}
-      {{- include "kubeProxyFiles" . | nindent 10 }}
+      {{- include "sshFiles" . | nindent 6 }}
+      {{- include "kubeProxyFiles" . | nindent 6 }}
+      - path: /etc/kubernetes/encryption/config.yaml
+        permissions: "0600"
+        contentFrom:
+          secret:
+            name: {{ include "resource.default.name" $ }}-encryption-provider-config
+            key: encryption
     preKubeadmCommands:
-      {{- include "kubeProxyPreKubeadmCommands" . | nindent 10 }}
+      {{- include "kubeProxyPreKubeadmCommands" . | nindent 6 }}
```

Comment:
it was wrong before? 🤔 how did it work then?

Reply:
I guess yaml is forgiving and indentation can be bigger than 2